marketplacetests/marketplace/regions/debug.py

```diff
@@ -21,6 +21,6 @@ def tap_back(self):
         self.marionette.find_element(*self._back_button_locator).tap()
 
     def select_region(self, region):
-        self.marionette.find_element(*self._region_select_locator).tap()
+        self.marionette.find_element(*self._region_select_locator).send_keys(region)
```

Comment:
I believe the issue here is that Marionette is not scrolling the element fully into view. This was fixed in [bug 937053](https://bugzilla.mozilla.org/show_bug.cgi?id=937053), which landed after the v2.0 branch was created. Perhaps a better workaround would be to force the element into view using:

```python
element = self.marionette.find_element(*self._region_select_locator)
self.marionette.execute_script('arguments[0].scrollIntoView(false);', [element])
element.tap()
```

I've tested this locally with success.